nav2_common/nav2_common/launch/parse_argument.py

```diff
@@ -0,0 +1,40 @@
+# Copyright (c) 2023 LG Electronics.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import sys
+
+from typing import Text
+
+class ParseArgument():
```

Comment:
Is this necessary? This **must** be something in launch ros, right? How does every other launch param do this?